### PR TITLE
Update contacts with current roles

### PR DESCRIPTION
**Pull request by ryzokuken**
Update the README file with the current roles filled in by the mentioned contacts.

/cc @hackygolucky @ashleygwilliams

P.S. I have already talked to Ashley about this and she seems fine with the change, awaiting response from Tracy, but considering that the election results were made public, it shouldn't be a problem, right?

### CODE REVIEW

**Comment by ryzokuken** [comment]
Everyone PTAL.

**Comment by ryzokuken** [comment]
I'm sorry I'm having to do this, but /cc @nodejs/community-committee

**Comment by WaleedAshraf** [comment]
This is related: https://github.com/nodejs/community-committee/pull/336

File: README.md [hunk 1]
```diff
@@ -20,9 +20,9 @@ Members of the TSC and CommComm shall operate under Lazy Consensus as a collabor
 For all votes, the vote must pass in both TSC and CommComm or else the motion fails. If an individual happens to be in both committees, then they would end up with two votes.
 
 ### Contacts for assistance
-- [@ashleygwilliams](https://github.com/ashleygwilliams) - **Ashley Williams**, Individual Membership Director
+- [@ashleygwilliams](https://github.com/ashleygwilliams) - **Ashley Williams**, Individual Membership Director Emeritus
```
[the remark this answers is not in the document]
Comment (MylesBorins) [reply README.md]: I think removing the line makes sense personally, we don't have Feros' contact information in here.

Alternatively we should make another section for Emeritus and include all past directors

**Comment by williamkapke** [comment]
@ryzokuken The Bylaws have not been changed yet to authorize the new CommComm Director role- so @ashleygwilliams is still a Director until that happens. But, a PR in optimistic preparation is helpful- it just shouldn't be merged yet.

**Comment by hackygolucky** [comment]
@williamkapke we should be able to merge this now, yes?

**Comment by mhdawson** [comment]
@hackygolucky I still think we need to resolve the questions about wether @ashleygwilliams should still be listed (as emeritus)

**Comment by williamkapke** [comment]
@hackygolucky Yes- The Bylaw changes to add the CommComm Director and remove one of the Individual Director seats was approved!

**Comment by Trott** [comment]
@nodejs/community-committee @nodejs/tsc This seems like a no-brainer to me, but it would be good to have a few more approvals before landing.

**Pull request merged**